integrations/salesforce/subscription.go

```diff
@@ -38,21 +40,18 @@ func (h handler) subscribe(clientID, orgID, instanceURL string, cid sdktypes.Con
 		return
 	}
 
-	ctx, client := h.initPubSubClient(cid, clientID, nil, "")
+	ctx, client := h.initPubSubClient(l, cid, clientID, "")
 	if ctx == nil || client == nil {
-		h.logger.Error("failed to create Salesforce client", zap.String("client_id", clientID))
+		l.Error("failed to create Salesforce client")
```

Comment:
Removing `zap.String("client_id", clientID)` was a mistake (I thought it's the connection ID, which is already logged). Will add back before mering.